app/controllers/api/v1/telegram_controller.rb

```diff
@@ -3,11 +3,22 @@ module V1
     class TelegramController < Api::V1::ApiController
       before_action :authenticate_user
 
+      # GET '/telegram'
+      def credentials
+        credentials = current_user.credentials.find_by(service: 'telegram')&.data
+        render json: {
+          telegram_chat_id: credentials['telegram_chat_id'] ||= '',
+          telegram_token: credentials['telegram_token'] ||= ''
+        }, status: :ok
+      end
+
+      # POST '/telegram'
       def telegram
-        if TelegramService.new(current_user, params).execute
-          render json: 'Telegram successfully created'
+        telegram = TelegramService.new(current_user, params)
+        if telegram.execute
+          render json: { message: 'Message sucessfully send!' }, status: :ok
         else
-          render json: 'Telegram not created'
+          render json: { error: telegram.error }, status: :ok
```

Comment:
why if some errors exists we send `status: :ok`?

Reply:
![photo_2019-08-31 17 55 23](https://user-images.githubusercontent.com/25904193/64074786-82156600-ccb8-11e9-8c12-3ea7985bb682.jpeg)